prisma/migrations/20210130094615_init/migration.sql

```diff
@@ -0,0 +1,42 @@
+-- CreateTable
+CREATE TABLE "Post" (
```

Comment:
Maybe I'm mistaken, but are you sure you need quote marks around the table name & column names?

Reply:
In the sense that, I think it's fine but I don't think you need them unless you need to escape certain things (e.g. a column named "serial" to not confuse its name with the data type)